conda_smithy/feedstock_content/.gitattributes

```diff
@@ -5,3 +5,18 @@
 meta.yaml text eol=lf
 build.sh text eol=lf
 bld.bat text eol=crlf
+
+# github helper pieces to make some files not show up in diffs automatically
+.azure-pipelines/* linguist-generated=true
+.circleci/* linguist-generated=true
```

Comment:
What about `.ci_support/*`? Was that intentionally left out?

Reply:
probably not, would be great to add

Reply:
I intentionally left it out so that rerenders that move pins are clear. But I can easily be swayed

Reply:
For example: https://github.com/conda-forge/python-confluent-kafka-feedstock/pull/19/files where a pin goes poof?